src/content/docs/style-guide/formatting/code-examples.mdx

```diff
@@ -127,7 +144,46 @@ To format inline code (like `maxSampleTimes` or `TransactionError`), surround th
 
 ## Multi-line code blocks [#code-block]
 
-To format one or more lines of code, insert three backticks ( ```` ``` ```` ) above and below your code. This essentially acts like the HTML `<pre>` tag.
+To format one or more lines of code, insert three backticks ( ```` ``` ```` ) above and below your code. This essentially acts like the HTML `<pre>` tag. 
+
+When you sandwich your code between three backticks, you need to consider both the code layout (indentation) and the syntax highlighting.
+
+### Code layout [#code-layout]
+
+As a technial writer you'll be given code snippets from a variety of languages. To make it easier for your readers to understand code snippets, make sure that lines are indented and grouped in a helpful way.
+
+#### Strict indentation [#strict]
```

Comment:
Can we restructure this to avoid the h4? Generally we try to avoid going deeper than h3s. I think you can do it by losing the `Code layout` header and turning `Strict indentation` and `Non-strict indentation` into h3s.

Reply:
@austin-schaefer I flattened this. Check out the latest preview.